gravity.sh

```diff
@@ -90,6 +90,16 @@ generate_gravity_database() {
   chmod g+w "${piholeDir}" "${gravityDBfile}"
 }
 
+update_gravity_timestamp() {
+  # Update timestamp when the gravity table was last updated successfully
+  output=$( { sqlite3 "${gravityDBfile}" <<< "INSERT OR REPLACE INTO info (property,value) values (\"updated\",cast(strftime('%s', 'now') as int));"; } 2>&1 )
```

Comment:
The `<<<` is not necessary due to `sqlite3` accepting the command as the second argument.

It still works this way and some other code does this as well, so still approving.